Practice2/src/partials/version-section.html

```diff
@@ -0,0 +1,14 @@
+<section class="page-version">
+  <picture>
+    <source media="(min-width: 1440px)" srcset="./assets/images/version-background.png">
+    <source media="(min-width: 768px)" srcset="./assets/images/version-background-tablet.png">
+    <source media="(min-width: 320px)" srcset="./assets/images/version-background-mobile.png">
+    <img src="./assets/images/version-background-mobile.png" alt="background" class="version-img">
+  </picture>
+  <div class="version-text-common">
+    <h1 class="version-heading">Get Your Free Version</h1>
```

Comment:
One page should have one h1 tag only. Please change to h3 → h6 following best practice.

Reply:
Fixed